front/types/Video.ts

```diff
@@ -1,12 +1,12 @@
-type sizes = '144' | '240' | '480' | '720' | '1080';
+export type videoSize = '144' | '240' | '480' | '720' | '1080';
 
 export interface Video {
   description: string;
   id: string;
   status: string;
   title: string;
   urls: {
-    jpeg: { [key in sizes]: string };
-    mp4: { [key in sizes]: string };
+    jpeg: { [key in videoSize]: string };
```

Comment:
rename it to thumbnails for coherence with the backend?

Reply:
done